src/backend/utils/text-parser.js

```diff
@@ -1,11 +1,23 @@
-const jsdom = require('jsdom');
+const puppeteer = require('puppeteer');
 
-const { JSDOM } = jsdom;
-
-// Simple function to convert html to text
-// Issue #75: https://github.com/Seneca-CDOT/telescope/issues/75
+/**
+ * Simple function to convert html to text
+ * @returns {string} a string of text parse from html
+ * Issue #75: https://github.com/Seneca-CDOT/telescope/issues/75
+ */
 exports.run = async function(html) {
```

Comment:
Change `html` to `htmlFragment` to help indicate that this is a portion of an HTML document, not the whole thing.